tests/rule-setup.js

```diff
@@ -30,6 +31,7 @@ describe('rules setup is correct', function() {
     const path = join(__dirname, '../tests/lib/rules');
     const files = readdirSync(path);
 
+    // eslint-disable-next-line node/no-deprecated-api
```

Comment:
Wasn't able to fix a few of these. `deepStrictEqual` fails here even though the diff/types of these arrays are the same.